frontend/src/components/explorer/MapViewer.vue

```diff
@@ -145,6 +148,7 @@ export default {
   },
   async created() {
     this.handleQueryParamsWatch = debounce(this.handleQueryParamsWatch, 100);
+    this.handleSidebarScroll = debounce(this.handleSidebarScroll, 300);
```

Comment:
Instead of debouncing, could `this.$refs.mapSidebar.scrollTop` be `watch`-ed instead?

Reply:
@e0 This is a smart solution for the problem of too narrow map listing. I personally think the folding of the Gene card should happen as soon as a user starts to scroll the map listing, so that change the waiting time from 300ms to e.g. 50ms would be better. But if you think it is better with a longer waiting time, I'm OK with that.

Reply:
> @e0 This is a smart solution for the problem of too narrow map listing. I personally think the folding of the Gene card should happen as soon as a user starts to scroll the map listing, so that change the waiting time from 300ms to e.g. 50ms would be better. But if you think it is better with a longer waiting time, I'm OK with that.

Haha @mihai-sysbio actually suggested 800ms. Maybe we can book a quick meeting to talk about this or maybe in tomorrow's stand-up if there is time.

Reply:
I had suggested 800ms _if_ there is no way of keeping this behaviour after removing the scroll reset code.

Reply:
>  I personally think the folding of the Gene card should happen as soon as a user starts to scroll the map listing,

My previous suggestion is in line with this @nanjiangshu - as soon as `this.$refs.mapSidebar.scrollTo` becomes non-zero, the card may be collapsed (with or without delay).

Reply:
> Instead of debouncing, could `this.$refs.mapSidebar.scrollTop` be `watch`-ed instead?

I'm not sure if that's possible but could look into it. What would the advantage be?

Reply:
`debounce` basically groups all the events that happened in a desired interval. What I am after is to not trigger as many events to start with. Perhaps `watch`-ing is not the right way..

Reply:
I have added [a commit](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/628/commits/3c4cd536b2379548c5e3bf9da2a1698e27c8e8e2) that folds the data panel immediately after scrolling down instead of using the debounce to wait. It is also conditionally bound so that the scroll does not trigger the scroll handler once the panel is folded.